docs/diagnostics/PX1056.md

```diff
@@ -0,0 +1,43 @@
+# PX1056
+This document describes the PX1056 diagnostic.
+
+## Summary
+
+| Code   | Short Description                          | Type  | Code Fix  | 
+| ------ | ------------------------------------------ | ----- | --------- | 
+| PX1056 | A PXGraph instance cannot be initialized inside the IsActive or IsActiveForGraph<TGraph> method. | Error | Unavailable | 
+
+## Diagnostic Description
+You cannot create a graph instance in the `IsActive` or `IsActiveForGraph<TGraph>` method because it can lead to a deadlock. 
```

Comment:
@EAndrosova may be we should mention that `IsActive` methods are declared in extensions? Something like this:
```suggestion
You cannot create a graph instance in the `IsActive` method of a graph or DAC extension and in the `IsActiveForGraph<TGraph>` method of a graph extension because it can lead to a deadlock. 
```
Although I'm not sure. What do you think?